src/microbe_stage/MicrobeStage.tscn

```diff
@@ -1591,6 +1598,7 @@ __meta__ = {
 [node name="HUDBottomBar" parent="MicrobeHUD/BottomLeft" instance=ExtResource( 16 )]
 margin_top = -43.0
 margin_bottom = -8.0
+SuicideButtonPath = null
```

Comment:
This seems like a suspicious change.

Reply:
I think in this case, this was just godot being godot, and making random changes to files for no reason. I will revert it though.